src/conversation/authorization.py

```diff
@@ -13,31 +13,30 @@
 BOT_GREETINGS_MESSAGE = (
     "Вы успешно начали работу с ботом. Меня зовут Женя Краб, "
     "я telegram-bot для экспертов справочной службы "
-    "'Просто спросить'. Я буду сообщать вам о новых заявках, "
+    "\"Просто спросить\". \nЯ буду сообщать вам о новых заявках, "
     "присылать уведомления о новых сообщениях в чате от пациентов "
-    "и их близких и напоминать о просроченных заявках. "
+    "и их близких и напоминать о просроченных заявках. \n"
     "Нам нравится, что вы с нами. Не терпится увидеть вас в деле! "
-    "Для начала, давайте настроим часовой пояс, чтобы вы получали "
-    "уведомления в удобное время."
 )
 BOT_QUESTON_YOU_ARE_EXPERT = (
-    "Привет! Этот бот предназначен только для экспертов справочной службы Просто спросить. Вы являетесь экспертом?"
+    "Привет! Этот бот предназначен только для экспертов справочной службы \"Просто спросить\". "
+    "Вы являетесь экспертом?"
 )
 BOT_QUESTON_WANT_BE_EXPERT = (
-    "Этот бот предназначен только для экспертов справочной службы ***'Просто спросить'***.\n"
-    "Хотите стать *нашим экспертом* 👨‍⚕️ и отвечать на заявки от пациентов и их близких?"
+    "Этот бот предназначен только для экспертов справочной службы \"Просто спросить\".\n"
+    "Хотите стать нашим экспертом и отвечать на заявки от пациентов и их близких?"
 )
 BOT_OFFER_ONLINE_CONSULTATION = (
-    'Этот бот предназначен только для экспертов справочной службы ***"Просто спросить"***.\n'
+    "Этот бот предназначен только для экспертов справочной службы \"Просто спросить\".\n"
     "Если у вас возникли вопросы об онкологическом заболевании, "
-    'заполните заявку на странице справочной службы ***"Просто спросить"***.'
+    "заполните заявку на странице справочной службы \"Просто спросить\"."
 )
 BOT_OFFER_FILL_FORM_FOR_FUTURE_EXPERT = (
-    "Мы всегда рады подключать к проекту новых специалистов!\nЗдорово, что вы хотите работать с нами 🤗.\n"
-    f"Заполните, пожалуйста, эту [анкету]({config.FORM_URL_FUTURE_EXPERT}) - нужно 15 минут.\n\n"
+    "Мы всегда рады подключать к проекту новых специалистов!\nЗдорово, что вы хотите работать с нами.\n"
+    f"Заполните, пожалуйста, эту [анкету]({config.FORM_URL_FUTURE_EXPERT})  (нужно 15 минут). "
     "Команда сервиса подробно изучит вашу заявку и свяжется с вами в течение недели, чтобы договориться о "
-    "видеоинтервью.\nПеред интервью мы можем попросить вас ответить на тестовый кейс, чтобы обсудить его на "
-    "встрече.\n\nЖелаем удачи 😊"
+    "видеоинтервью.\nПеред интервью мы можем попросить вас ответить на тестовый кейс,"
+    " чтобы обсудить его на встрече."
```

Comment:
Traceback (most recent call last):
  File "/home/ealmina/code/ask_nenaprasno_bot/src/core/send_message.py", line 82, in reply_message
    await message.reply_markdown_v2(text=text_to_markdown(text), reply_markup=reply_markup)
  File "/home/ealmina/code/ask_nenaprasno_bot/.venv/lib/python3.10/site-packages/telegram/_message.py", line 920, in reply_markdown_v2
    return await self.get_bot().send_message(
  File "/home/ealmina/code/ask_nenaprasno_bot/.venv/lib/python3.10/site-packages/telegram/_bot.py", line 236, in decorator
    result = await func(*args, **kwargs)
  File "/home/ealmina/code/ask_nenaprasno_bot/.venv/lib/python3.10/site-packages/telegram/_bot.py", line 649, in send_message
    return await self._send_message(  # type: ignore[return-value]
  File "/home/ealmina/code/ask_nenaprasno_bot/.venv/lib/python3.10/site-packages/telegram/ext/_extbot.py", line 254, in _send_message
    result = await super()._send_message(
  File "/home/ealmina/code/ask_nenaprasno_bot/.venv/lib/python3.10/site-packages/telegram/_bot.py", line 352, in _send_message
    result = await self._post(
  File "/home/ealmina/code/ask_nenaprasno_bot/.venv/lib/python3.10/site-packages/telegram/_bot.py", line 315, in _post
    return await request.post(
  File "/home/ealmina/code/ask_nenaprasno_bot/.venv/lib/python3.10/site-packages/telegram/request/_baserequest.py", line 162, in post
    result = await self._request_wrapper(
  File "/home/ealmina/code/ask_nenaprasno_bot/.venv/lib/python3.10/site-packages/telegram/request/_baserequest.py", line 325, in _request_wrapper
    raise BadRequest(message)
telegram.error.BadRequest: Can't parse entities: character '(' is reserved and must be escaped with the preceding '\'